src/main/java/bio/terra/janitor/app/configuration/AzureConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package bio.terra.janitor.app.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "janitor.azure")
+public class AzureConfiguration {
```

Comment:
I'm unsure about this structure. This would have a Janitor configured to clean up a specific managed app installation in a tenant.

What will we do when there is more than one installation in an environment?

Reply:
Technically Janitor here has the identity of a managed app _publisher_. So it can clean up deployments of that managed app across tenants/subscriptions. The tenant/subscription/mrg is included in the `CloudResourceUid` message.

Even if we're working with multiple managed app _definitions_ it should work I think -- we would just need to add the Janitor `clientId` in the Terra tenant to the managed app publisher group.

Reply:
Got it, thanks